src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -441,6 +440,14 @@ private async Task<List<string>> GetOrderedFilesToRestore()
             var fullBackupPath = GetBackupPath(backupPath);
             using (var zip = await GetZipArchiveForSnapshot(fullBackupPath))
             {
+                var voronDataDirectory = new VoronPathSetting(RestoreFromConfiguration.DataDirectory);
```

Comment:
voronDataDirectory  => restorePath?

Reply:
Logic was previously more complex:

```
                var baseDataDirectory = ServerStore.Configuration.Core.DataDirectory.FullPath;

                var destinationPath = string.IsNullOrEmpty(restoreBackup.DataDirectory) == false
                    ? new PathSetting(restoreBackup.DataDirectory, baseDataDirectory).FullPath
                    : RavenConfiguration.GetDataDirectoryPath(ServerStore.Configuration.Core, restoreBackup.DatabaseName, ResourceType.Database);
```

We didn't miss something?

Reply:
VoronPathSetting? Why not PathSetting?

Reply:
> Logic was previously more complex:
> 
> ```
>                 var baseDataDirectory = ServerStore.Configuration.Core.DataDirectory.FullPath;
> 
>                 var destinationPath = string.IsNullOrEmpty(restoreBackup.DataDirectory) == false
>                     ? new PathSetting(restoreBackup.DataDirectory, baseDataDirectory).FullPath
>                     : RavenConfiguration.GetDataDirectoryPath(ServerStore.Configuration.Core, restoreBackup.DatabaseName, ResourceType.Database);
> ```
> 
> We didn't miss something?

in `SnapshotRestore` method the restore path is voronDataDirectory I dont see any point to use the logic which we had in handler.



> VoronPathSetting? Why not PathSetting?

this is what it was I just reorder it https://github.com/ravendb/ravendb/pull/11023/files#diff-e08d344e1eb26f22609c114ab3008af6L483